garak/_config.py

```diff
@@ -158,11 +158,12 @@ def load_config(
 
 def parse_plugin_spec(
     spec: str, category: str, probe_tag_filter: str = ""
-) -> List[str]:
+) -> tuple[List[str], List[str]]:
```

Comment:
I would usually prefer a typed single object returned, however this method is currently only consumed by `cli.py`, to limit impacts a tuple was returned for `fully resolved accepted` and `rejected` values from the spec offered.

Reply:
Have you considered returning a dict? Or raising an exception in the case that there are unknown clauses, and returning the leftover pieces in the exception, for the calling method to handle however it chooses? Maybe gating exception raising/ignoring on a named method var here?